src/base/bittorrent/session.cpp

```diff
@@ -472,6 +472,7 @@ Session::Session(QObject *parent)
     , m_peerTurnover(BITTORRENT_SESSION_KEY(u"PeerTurnover"_qs), 4)
     , m_peerTurnoverCutoff(BITTORRENT_SESSION_KEY(u"PeerTurnoverCutOff"_qs), 90)
     , m_peerTurnoverInterval(BITTORRENT_SESSION_KEY(u"PeerTurnoverInterval"_qs), 300)
+    , m_storageType(BITTORRENT_SESSION_KEY(u"StorageType"_qs), StorageType::Default)
```

Comment:
Couldn't it be better to use the name "Disk IO type"? "Storage" is already used for another purposes.

Reply:
Well, yeah, I admin it could be confusing, but the libtorrent term of this stuff is "Custom_Storage", so I'd like to keep it here.

Reply:
I would still prefer that we use a consistent (and less confusing) terminology rather than blindly adhere to libtorrent's one.
Let's wait for other opinions.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/16895#discussion_r853712540